examples/scripts/os-https.cli

```diff
@@ -1,9 +1,8 @@
 #SSL
-if (outcome == success) of /core-service=management/security-realm=ApplicationRealm/server-identity=ssl:read-resource
-  /core-service=management/security-realm=ApplicationRealm/server-identity=ssl:remove
-end-if
-/core-service=management/security-realm=ApplicationRealm/server-identity=ssl:add(keystore-path="${keystore.path}", keystore-password="${keystore.password}")
+/subsystem=elytron/key-store=keystore:add(path="${keystore.path}", credential-reference={clear-text=${keystore.password}})
```

Comment:
@darranl , the switch to use elytron reveals a problem in the approach. The keystore must exists or the operation will not apply (even in admin-mode). With the legacy security-realm, the path to the keystore can be un-existing.
For Openshift, the user is going to set some  pathes that would only exist in the POD. Could we have an expression (such as ${env.MY_KEYSTORE_PATH}) for the keystore path?

Reply:
@jfdenise How about if we add the attribute "required=false" ?

Reply:
Adding required: /subsystem=elytron/key-store=keystore:add(required=false, path="${keystore.path}", credential-reference={clear-text=${keystore.password}})
gives an error: 
"failure-description" => {"WFLYCTL0080: Failed services" => {"org.wildfly.security.key-store.keystore" => "WFLYELY00004: Unable to start the service.
[ERROR]     Caused by: org.jboss.msc.service.StartException in anonymous service: WFLYELY00022: KeyStore file '/etc/wf-secrets/keystore.jks' does not exist and required."}},